Assignment1/src/test/java/AssignmentMain.java

```diff
@@ -0,0 +1,107 @@
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.MatcherAssert.assertThat; 
+import static org.hamcrest.Matchers.*;
+import io.restassured.path.json.JsonPath;
+
+public class AssignmentMain {
```

Comment:
Try coming up with a better class name. What work does this represent? Is this the starting point for running all the tests? If so then a name like `AutomateReqResTest` may make sense.

Reply:
Added better class name